vendor/go.etcd.io/etcd/etcdserver/errors.go

```diff
@@ -39,6 +39,7 @@ var (
 	ErrKeyNotFound                = errors.New("etcdserver: key not found")
 	ErrCorrupt                    = errors.New("etcdserver: corrupt cluster")
 	ErrBadLeaderTransferee        = errors.New("etcdserver: bad leader transferee")
+	ErrMemberRemoved              = errors.New("the member has been permanently removed from the cluster")
```

Comment:
Since we're hoping to have this change pulled upstream, lets use their convention for error messages.

```suggestion
	ErrMemberRemoved              = errors.New("etcdserver: member has been permanently removed from the cluster")
```